draft-kcs-tvr-requirements.xml

```diff
@@ -333,6 +333,72 @@ Likewise, the assigning of an address property at one layer does not imply the p
       <t>TBD</t>
     </section>
 
+     <section anchor="requirements-summary">
+      <name>Requirements Summary</name>
+
+      <section title="Support the Identification and Advertisement of Node Attribute Changes">
+        <t>In Time-Variant Routing, scheduling of the system are as expected.
+        In practical situations, however, the attributes of nodes can be
+        converted back and forth between Time-Variant and Non-Time-Variant
+        nodes. For example, a mobile phone can be considered a
+        Non-Time-Variant node in the network when charging. After unplugging
+        the power cord, it can be considered a Time-Variant-Node because it
+        starts to be powered by batteries.</t>
```

Comment:
This was something discussed at the last IETF around terminology; a node (or other entity) isn't considered as "not time variant" because it doesn't have any scheduled changes. The network as a whole is time-variant and each entity can have an arbitrarily complex schedule for its properties, including a trivial "no changes" non-schedule.